src/types.ts

```diff
@@ -6,22 +6,40 @@ export interface Options {
    * Relative paths to the directory to search for components.
    * @default 'src/components'
    */
-  dirs: string | string[]
+  include_dirs: string | string[]
+
+  /**
+   * Directory names that are ignored as Component names.
+   * Only works if Foldernames are allowed
+   * @default "['global', 'partials']"
+   */ 
+  namespaces: string[]
```

Comment:
`namespaces` might be too general for this feature. Maybe

- globalNamespaces
- globalPrefixes

Reply:
globalNamespaces sounds better here!